src/python/WMCore/REST/Format.py

```diff
@@ -244,6 +244,7 @@ def stream_chunked(self, stream, etag, preamble, trailer):
                 trailer = None
                 raise
             finally:
+                print("ERROR, json.dumps failed to serialize %s, type %s" % (obj, type(obj)))
```

Comment:
If we don't catch the exception raised, finally block will re-raise it. So I think we should create a general except block, print the error message and raise it again.
Otherwise, this print will just show up for success or exception cases.